src/plans/D3MAaveDaiPlan.sol

```diff
@@ -154,19 +145,16 @@ contract D3MAaveDaiPlan is ID3MPlan {
         // Do inverse calculation of interestStrategy
         uint256 variableRateSlope1 = interestStrategy.variableRateSlope1();
         uint256 targetUtil;
-        if (targetInterestRate > _add(base, variableRateSlope1)) {
+        if (targetInterestRate > base + variableRateSlope1) {
             // Excess interest rate
             uint256 r = targetInterestRate - base - variableRateSlope1;
-            targetUtil = _add(
-                            _rdiv(
-                                _rmul(interestStrategy.EXCESS_UTILIZATION_RATE(), r),
-                                interestStrategy.variableRateSlope2()
-                            ),
-                            interestStrategy.OPTIMAL_UTILIZATION_RATE()
-                        );
+            targetUtil = _rdiv(
+                            _rmul(interestStrategy.EXCESS_UTILIZATION_RATE(), r),
+                            interestStrategy.variableRateSlope2()
+                        ) + interestStrategy.OPTIMAL_UTILIZATION_RATE();
         } else {
             // Optimal interest rate
-            targetUtil = _rdiv(_rmul(_sub(targetInterestRate, base), interestStrategy.OPTIMAL_UTILIZATION_RATE()), variableRateSlope1);
+            targetUtil = _rdiv(_rmul((targetInterestRate - base), interestStrategy.OPTIMAL_UTILIZATION_RATE()), variableRateSlope1);
```

Comment:
refactor to be multiline

Reply:
Also consider removing the brackets around `targetInterestRate - base`